README.md

```diff
@@ -65,6 +65,20 @@ steps:
       release-rules:
         '[{ "release": "patch", "type": "build" }, { "release": "patch", "type":
         "chore(deps)" }, { "release": "patch", "type": "chore(deps-dev)" }]'
+      # Append additional semantic release commit analyzer rules to the default set of rules already included in the action.
+      # Default rules:
+      # [
+      #  { "release": "patch", "type": "build" },
+      #  { "release": "patch", "type": "chore" },
+      #  { "release": "patch", "type": "ci" },
+      #  { "release": "patch", "type": "docs" },
+      #  { "release": "patch", "type": "improvement" },
+      #  { "release": "patch", "type": "refactor" },
+      #  { "release": false, "subject": "*\\[skip release\\]*" },
+      # ]
+      # Note: these do not override the default rules. You cannot configure 'release-rules' if you wish to append rules.
```

Comment:
Maybe for clarity rephrase to: `cannot be used in conjunction with 'release-rules'` and add to the first sentence.

Reply:
Much clearer way to phrase it thanks.